src/couch_replicator/src/couch_replicator_auth_session.erl

```diff
@@ -210,6 +210,8 @@ init_state(#httpdb{} = HttpDb) ->
                     {ok, HttpDb1, State1};
                 {error, {session_not_supported, _, _}} ->
                     ignore;
+                {error, {session_requires_valid_user, _, _}} ->
+                    ignore;
```

Comment:
ah, ok, so even if we have the credentials we need, we don't even try using them, we're just going to fail out on _session handling and fail back to per-request auth?